### DISCUSSION TITLE
("Standalone Pico4") + Termux = PicoRoider ?

### DISCUSSION THREAD

**Discussion by Uttis** [termux/termux-app#4070] [Ideas]
I would like try use Termux terminal to optimize the HMD and have the option of go superuser and root the HMD,
i have alot of ideas but need to be honest, i am verified noob when it comes to coding and moderate.
the Termux is incredible tool set for software development and easy-to-use what i can tell!

but would need to get abit help with how i would be able to use a backdoor to Root pico and unlock it to make it limitless without any PC or ADB..

i would love to get you guys teach me how to make my left brain work with the right toolbox and deploy useful applications to use extra..

**Comment by agnostic-apollo** [comment]
We do not provide support for rooting devices, check https://xdaforums.com/ instead.
